guardian/scribe-plugin-noting
# Toggle note tag names

This PR adds the ability to click a note and toggle it's tag name i.e. `gu-flag` > `gu-correct`. This has to be configured during plugin init.

### QA
- Create a flag (F9), then click the give flag.  The tag name should have changed to `gu-correct`.
